README.md

```diff
@@ -13,20 +13,29 @@ Maintain SKU specific package content such as templates and other static files.
 
 * Build SKU package
 
+  * Create sku pack with SKU build-package tool and on the root folder run:
+
   ```
-  $ ./build-package.bash <sku_pack_directory> <subname>
+  $ ./build-package.bash <sku_pack_name> <subname>
+  Note: for below command <sku_pack_name> must be exactly the same with one of the node names listed on the root folder of on-skupack,
+  e.g. it can be quanta-d51-1u, quanta-t41,dell-r630, etc.<subname> can be any name you like. A {sku name}.tar.gz will be created in
```

Comment:
what about replaceing "{sku name}" with "{sku_pack_name}-{subname}"? (not sure whether correct). otherwise readers don't know what {sku name} would be expected.